synphot/tests/test_wrapper.py

```diff
@@ -0,0 +1,48 @@
+from synphot.models import BlackBody1D
+from synphot.spectrum import SourceSpectrum, SpectralElement
+from synphot.wrapper import Observe
+import astropy.units as u
+import numpy as np
+
+
+class TestObserve:
```

Comment:
It's not strictly necessary, but one style point: it's more explicit to write `class TestObserve(object)`, implying that `TestObserve` inherits from `object`.